torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -248,6 +250,44 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
 
                     graph.erase_node(node)
 
+    # Now do one more pass that specializes all symfloats we didn't manage
+    # to tensorify away.
+    nodes = list(graph.nodes)
+    for i, node in enumerate(nodes[:-1]):
+        with graph.inserting_before(
+            nodes[i + 1] if node not in placeholders else first_non_placeholder
+        ):
+            args = []
+            transform = False
+            for a in node.args:
+                if (
+                    isinstance(a, fx.Node)
+                    and "val" in a.meta
```

Comment:
I had to be more defensive here b/c tensor constants don't have "val" set in meta

eg.

```
b = torch.tensor(1.0, dtype=torch.double, device=self.device)
```

`python test/inductor/test_torchinductor.py CpuTests.test_clamp_type_promotion_cpu`

doesn't have "val" set in meta and results in a KeyError